spec/unit/renderer_spec.rb

```diff
@@ -36,7 +40,7 @@ class BarBazScope < Machinery::Object
 end
 
 class BarBazRenderer < Renderer
-  def do_render
+  def content(description)
```

Comment:
Unused method argument - `description`. If it's necessary, use `_` or `_description` as an argument name to indicate that it won't be used. You can also write as `content(*)` if you want the method to accept any arguments but don't care about them.